components/org.wso2.carbon.identity.application.authenticator.magiclink/src/main/java/org/wso2/carbon/identity/application/authenticator/magiclink/MagicLinkAuthenticator.java

```diff
@@ -697,4 +700,20 @@ private void setAuthParamsForIdfInitiatedFromMagicLink(AuthenticatorData authent
         authenticatorParamMetadataList.add(usernameMetadata);
         authenticatorData.setAuthParams(authenticatorParamMetadataList);
     }
+
+    /**
+     * Get the multi option URI query params.
+     *
+     * @param request HttpServletRequest.
```

Comment:
Shall we add the missing return statement description to this method description?

Reply:
Fixed with https://github.com/wso2-extensions/identity-local-auth-magiclink/pull/53